### ISSUE TITLE
Pipeline rules: == for strings containing double quotes fails to match

### ISSUE THREAD

**Issue by AGBrown** [Graylog2/graylog2-server#2688]
Within a pipeline rule, when matching strings in messages where the string contains `"` characters, the following will evaluate as `true`:

`$message.message == "Something \"quoted\" failed"`

## Expected Behavior
It isn't mentioned in the documentation, but from the context help in the rule editor it appears that double-quote characters should be escaped with backslash in strings in the rule editor.

It would therefore follow that both of the following should evaluate as `true`:

1. `$message.message == "Something \"quoted\" failed"`
1. `to_bool(regex("^Something \"quoted\" failed$", to_string($message.message)).matches)`

## Current Behavior
The string equality test seems to evaluate as `false` when the string contains double quotes.

## Steps to Reproduce (for bugs)

This rule works:

```
rule "test rule 1"
when
  to_bool(regex("^Something \"quoted\" failed$", to_string($message.message)).matches)
  && has_field("is_test")
then
  set_field("field_test_rgx", true);
end
```

This rule does not, but should do:

```
rule "test rule 2"
when
  $message.message == "Something \"quoted\" failed"
  && has_field("is_test")
then
  set_field("field_test_eq", true);
end
```

## Context
Trying to match strings. Strings not containing quotes match fine, but strings containing quotes do not.

## Your Environment

* Graylog Version: Graylog 2.0.3 (f07c170)

**Comment by edmundoa** [comment]
Hi,

I was trying to reproduce this issue in Graylog 2.1.0-beta.3 and I couldn't. I think it got fixed in Graylog2/graylog-plugin-pipeline-processor#47. This will be fixed in 2.1.0, please take a look the current beta if you want to ensure it is working for you too.

Thank you!

**Comment by edmundoa** [comment]
By the way, here's the link if you want to download the beta: https://www.graylog.org/blog/65-announcing-graylog-2-1-0-beta-3
